### PR TITLE
Update Specification for v0.23.0

### PR DESCRIPTION
**Pull request by alindima**
## Reason for This PR

The documentation was out of date.

## Description of Changes

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

**Review by serban300 — DISMISSED** [review]

File: FAQ.md
Comment (raduiliescu) [inline FAQ.md:41]: Firecracker support for Arm CPUs is released as developer preview.

Comment (alindima) [reply FAQ.md]: done

**Review by serban300 — DISMISSED** [review]

File: FAQ.md
Comment (raduiliescu) [inline FAQ.md:39]: The answer here should be the same as in this discussion https://github.com/firecracker-microvm/firecracker/pull/2194. I'll submit only the i3 to m5 change and let that discussion to finish and add this part to that commit

Comment (alindima) [reply FAQ.md:39]: As @sandreim suggested, I think we could simply link here the `Supported platforms` heading from `Readme.md`. WDYT? This way, we wouldn't have to duplicate that paragraph and would only have one place to maintain it.

Comment (alindima) [reply FAQ.md:39]: I will drop the commit modifying the FAQ and wait until #2194 is merged.

**Comment by alindima** [comment]
removed the commit modifying the FAQ. I'll be creating another PR for it once #2194 is merged. @raduiliescu @serban300 review again

**Review by serban300 — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

**Pull request merged**
